From aa5a66158970573745dcad7a8ff53c06f1224cf5 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 7 Apr 2014 09:03:11 -0400
Subject: [PATCH] Fix a build error that's been hiding.

Related: rhbz#1084547

Signed-off-by: Peter Jones <pjones@fedoraproject.org>
---
 memdisk/setup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/memdisk/setup.c b/memdisk/setup.c
index 3f69cd3..f485009 100644
--- a/memdisk/setup.c
+++ b/memdisk/setup.c
@@ -119,7 +119,7 @@ static const char *getcmditem(const char *what)
  */
 #define UNZIP_ALIGN 512
 
-extern void _end;		/* Symbol signalling end of data */
+extern uint8_t _end;		/* Symbol signalling end of data */
 
 void unzip_if_needed(uint32_t * where_p, uint32_t * size_p)
 {
-- 
1.9.0

